content/en/tracing/service_catalog/setup.md

```diff
@@ -17,23 +17,20 @@ further_reading:
 
 Datadog Service Catalog includes all discovered services from APM, USM, and RUM by default. If you are using any of these products, your catalog is pre-populated with entries. You can add service metadata to these entries through the Datadog UI, [API][1], or use automated pipelines through the [GitHub integration][11] or [Terraform][2].
 
-## For services with existing telemetry data (APM/USM/RUM)
+## Add service definition metadata for services
 
-Any service that sends tracing data to Datadog APM, USM, or RUM is automatically listed in the Service Catalog. Each unregistered service shows a gray check mark when you hover over it.
+1. On the [Service Catalog][10] page, click **Setup & Config**. The **Manage Entries** tabs shows you how many services are without metadata.
 
-To register the service and add ownership information, related links such as runbooks, and links to source code repositories, [update the service definition][1].
+2. Click **Create New Entry**.
 
-Navigate to [**APM** > **Service Catalog**][10] and click the [**Setup & Config** tab][3] to get help with forming valid JSON to post with the API.
+3. Specify which service you are adding metadata to. This can be name of a service already listed in the Service Catalog, or it can be the name of a service not sending any data.
```

Comment:
```suggestion
3. Specify which service you are adding metadata to. This can be the name of a service already listed in the Service Catalog, or it can be the name of a service not sending any data.
```